kernel-laws/src/main/scala/cats/kernel/laws/PartialOrderLaws.scala

```diff
@@ -0,0 +1,55 @@
+package cats.kernel.laws
+
+import cats.kernel.PartialOrder
+
+trait PartialOrderLaws[A] extends EqLaws[A] {
+  override implicit def E: PartialOrder[A]
+
+  def reflexitivity(x: A): IsEq[Boolean] =
+    E.gteqv(x, x) <-> true
```

Comment:
the original test uses `?<=`, i.e. `E.lteqv`,  I am not sure if it matters. (mabye we can have both `lteqv` and `gteqv`?)